media/js/app/assetmgr/collection.js

```diff
@@ -482,7 +482,7 @@ CollectionList.prototype.updateSwitcher = function () {
     Mustache.update("switcher_collection_chooser", self.switcher_context, { parent: self.parent });
 
     // hook up switcher choice owner behavior
-    jQuery(self.el).find("a.switcher-choice.owner").unbind('click').click(function (evt) {
+    jQuery(self.el).find("a.switcher-choice.owner").off('click').on('click', function (evt) {
```

Comment:
The bind/unbind behavior was originally written as the client-side template/dom element replacement caused issues. A handler attached to a non-existent element would fire incorrectly. Theoretically, the unbind might be eliminated now and replaced with a .on statement following a delegated event pattern.